### PR TITLE
setup partial address test

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@alicoding r? use "?test=address-test" to test.

File: components/address-input.jsx [hunk 1]
```diff
@@ -1,6 +1,256 @@
 import React from 'react';
 import IntlMixin from 'react-intl';
 
+var CountrySelect = React.createClass({
```
Comment (ScottDowne) [inline components/address-input.jsx]: I'm just making CountrySelect a reusable thing internally. Mean copy and pasting a semi-large chunk of code.

File: components/address-input.jsx [hunk 2]
```diff
@@ -637,4 +656,77 @@ var AddressInput = React.createClass({
...
+          </div>
+        </div>
+      </div>
+    );
+  }
+});
+
+module.exports = {
+  FullAddress: FullAddress,
+  PartialAddress: PartialAddress
```
Comment (ScottDowne) [inline components/address-input.jsx]: Returning the two types of address, I didn't get this to work with an array, the blocks themselves are not boiled down enough, so I didn't bother.

**Comment by alicoding** [comment]
tests are failing

**Comment by ScottDowne** [comment]
Fixed.

**Comment by alicoding** [comment]
r+

**Pull request merged**